tests/test_interpreters.py

```diff
@@ -63,10 +58,10 @@ class envconfig:
 
     p = tox_get_python_executable(envconfig)
     assert p == py.path.local(sys.executable)
-    for ver in "2.7 3.4 3.5 3.6".split():
-        name = "python%s" % ver
-        if sys.platform == "win32":
-            pydir = "python%s" % ver.replace(".", "")
+    for major, minor in tox.PYTHON.CPYTHON_VERSION_TUPLES:
+        name = "python%s.%s" % (major, minor)
```

Comment:
format instead of %? all throughout this file this is true

Reply:
I'll leave this for another PR. This one is getting ginormous already and Id rather merge it this weekend and publish a minor update as rc and then soon the real thing (without any other features or bugfixes to make sure that if new problems occur they are purely due to the spring cleaning).